components/Tabs/Tabs.js

```diff
@@ -2,31 +2,30 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element = element;
+
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = document.querySelector(`.tabs-link[data-tab='${element.dataset.tab}']`);
+
```

Comment:
I know you have this part locally as we went through it during evening standup so I'll wait to comment until you push the changes up